build.ps1

```diff
@@ -262,6 +262,13 @@ function cleanExtension([string] $bitness) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    if ($bitness -eq "64bit") {
+        $keepRuntimes = @('win', 'win-x64', 'win10-x64')
+    } elseif ($bitness -eq "32bit") {
+        $keepRuntimes = @('win', 'win-x86', 'win10-x86')
+    }
+    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes | Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
It just occurred to me, this path is specific to the powershell worker, but this function is used in other cases too, would that be a problem?

I noticed the `silentlycontinue` would hide all not-found errors, but the `Get-ChildItem` used in this function don't have erroraction declared ...

Reply:
And by the way, can you please test the ps worker with those 3 runtime folders to see if ps function app works fine?

Reply:
```suggestion
    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\powershell\runtimes" -Exclude $keepRuntimes -ErrorAction SilentlyContinue |
        Remove-Item -Recurse -Force -ErrorAction SilentlyContinue
```

will change this...

Also the PSWorker does work with only those 3 runtime folders. I ran a smoke test.

Reply:
> Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary"

What about the `Get-ChildItem` call above in this function? Do we need `-EA` for that too?